validator/postgresql_log_checkpoints_test.rego

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+package templates.gcp.GCPPostgreSQLCheckpointsConstraintV1
+
+import data.validator.gcp.lib as lib
+import data.validator.test_utils as test_utils
+
+import data.test.fixtures.postgresql_log_checkpoints.assets.no_settings as fixture_no_settings
+import data.test.fixtures.postgresql_log_checkpoints.assets.no_violations as fixture_no_violation
+import data.test.fixtures.postgresql_log_checkpoints.assets.violations as fixture_violation
+
+import data.test.fixtures.postgresql_log_checkpoints.constraints as fixture_constraint
+
+template_name := "GCPPostgreSQLCheckpointsConstraintV1"
+
+#1. postgresql with correct key
+test_postgresql_log_checkpoints_with_correctkey {
+	expected_resource_names := {"//cloudsql.googleapis.com/projects/my-test-project/instances/tf-pg-ha-62380f9c-no-violation"}
```

Comment:
Are you expecting this fixture to raise a violation? If so, it should *not* be called "with_correctkey" and you *definitely* should not expect a resource called "no-violation" to raise a violation. Either your naming or logic is wrong.

Reply:
@morgante This case(postgresql with correct key) don't raise a violation in our case. So I have removed the test case since it's not needed. Violated test cases alone is attached. Hope this is Ok. Please review it.